### ISSUE TITLE
Sidebar fading

### ISSUE THREAD

**Issue by ChillyPigeon742** [MarkChan0225/RoundedDiscord#55]
As of recently (might have been a discord update or from vencord I dont really know) the sidebar has been like this where it fades out. I use vencord I should add that bit of info other than that I dont really know how this happened. I also should add that it fades out only at the explore button and then after that it continues, well that's in just the normal light/dark theme. When using nitro themes the sidebar completely dissapears from the start of the explore button. Below I attached some images of this

Light Theme:
![image](https://github.com/user-attachments/assets/a2f5be15-b326-4fd9-9fdb-97f598ad1106)

Dark Theme:
![image](https://github.com/user-attachments/assets/89e22545-43ac-443d-8231-8e308ea4d05b)

Nitro Theme (Midnight Blurple):
![image](https://github.com/user-attachments/assets/b65ed7e5-320f-4c9d-ac1b-2f381c5cab8b)

**Comment by ChillyPigeon742** [comment]
Yea so I only realized this after I posted that if you have enough servers to where the list is filled then the explore button is at the bottom as normal but if not then it will be up there and glitched. Also i've tried to to disable all the other themes I have and it is still the same.

**Comment by MarkChan0225** [comment]
Yup, that is a strange bug. I have posted a fix for it. However, I don't really the cause of this bug. Anyway, thanks for your report!

**Comment by ChillyPigeon742** [comment]
No problem, also sorry to bother you like this but I have restarted my discord and the update has only just gotten rid of the fading and put the explore button on the bottom instead of just keeping the explore button there and bringing back the rounded bar. Also I have found a couple things that arent rounded. Below I attached some images of these. There is also more i just cant provide images of it since ive done all the quests but basically everything from the quest tab above your profile card on the bottom left to the pop out of quests on user profiles basically everything quests related isnt rounded.

Side Bar:
![image](https://github.com/user-attachments/assets/30288ce8-d9e5-4601-80af-803cf0452265)

Activity on the profile not being rounded:
![image](https://github.com/user-attachments/assets/a636ec7b-70d1-485c-9cd6-0a0501a055ac)

**Comment by MarkChan0225** [comment]
```
.footer_d71945 {
    background: unset;
    margin-top: -8px;
}```

Put this script in your customcss will fix the problem of the sidebar. I don't think I will put this into the theme becuase the look now is not bad.

**Comment by MarkChan0225** [comment]
Activity fixed

**Comment by ChillyPigeon742** [comment]
Ok thanks I will try it when I am on my computer

**Comment by deadmau1** [comment]
![image](https://github.com/user-attachments/assets/c66bc6a2-2c9d-473a-9339-f81c78062a11)

Hey possible to make this rounded too??

**Comment by ChillyPigeon742** [comment]
Ok so I have tried it and the activity is now rounded but about the custom css it only makes the explore button go up which is how it should be but the side bar below it is still gone

![image](https://github.com/user-attachments/assets/cfc969ec-6577-4ad5-a9b2-69f4ef1e2c98)

**Comment by ChillyPigeon742** [comment]
Also I should note I have tried to add a lot of servers to test and the problem is the explore button is standing seperate from the side bar when i scroll up and down the explore button stays down instead of moving in the list like it was before

![image](https://github.com/user-attachments/assets/8a6f572f-8f8f-4fd1-9b1f-48419d1b6734)

**Comment by ChillyPigeon742** [comment]
this solves the problem a bit more but still not fully

> ```
> .footer_d71945 {
>     background: unset;
>     margin-top: 0px;
> }
> 
> .scroller_fea3ef.none_c49869.scrollerBase_c49869 {
>     border-radius: var(--radius) !important;
> }
> ```
> 
> or
> 
> ```
> .footer_d71945 {
>     background: var(--background-secondary);
>     border-bottom-left-radius: var(--radius) !important;
>     border-bottom-right-radius: var(--radius) !important;
>     margin-top: -8px;
> }
> ```

it leaves it like this when i use this code and my server list is full:

.footer_d71945 {
    background: var(--background-secondary);
    border-bottom-left-radius: var(--radius) !important;
    border-bottom-right-radius: var(--radius) !important;
    margin-top: -8px;
}

![image](https://github.com/user-attachments/assets/f1c1cf90-aa21-4994-984b-b1b1ca48abcc)

the problem is the explore button still doesnt scroll in the list it stays at the bottom

and also when my server list isnt full its like this:
![image](https://github.com/user-attachments/assets/1e526b9d-db0a-4487-b639-972b613c3d44)

the side bar doesnt extend to the bottom

also back to what i said about the quests not being rounded here are some screenshots as i can actually provide them now

when u click on a person in the members list playing a game that has a quest:
![image](https://github.com/user-attachments/assets/445fdc7a-04a8-4337-8481-8e6b243f68c3)

the quest popout in a server when not extended:
![image](https://github.com/user-attachments/assets/ef481245-7f89-4768-ae41-987c796e718d)

when extended  in a server (also notice how the channel list doesnt go up like it does with other things such as the spotify controls plugin):
![image](https://github.com/user-attachments/assets/8218b290-8037-4649-950b-94010b753ab4)

when extended in a server and chosen platform:
![image](https://github.com/user-attachments/assets/a58f7f6e-53b6-4d69-ac68-47e55b5166ca)

quest poput in dms:
![image](https://github.com/user-attachments/assets/e1593a7b-f9e4-4130-9a13-249ede6ecb98)

when extended in dms (same as in a server):
![image](https://github.com/user-attachments/assets/69e4189c-fbe5-4477-904a-651b78d69a94)

when extended in dms and chosen platform:
![image](https://github.com/user-attachments/assets/5f584d24-b213-4e4a-a4a5-ca01609bb2dd)

also should add that i found out the explore tab at the bottom is not rounded:
![image](https://github.com/user-attachments/assets/694f9866-6466-49cf-80ff-9d972cb214dc)

quest popout in explore tab (notice how the list doesnt go up at all contrary to the channel list in servers and dms):
![image](https://github.com/user-attachments/assets/d7246aac-3746-42e6-baa7-b3486ed50b3a)

when extended in explore tab:
![image](https://github.com/user-attachments/assets/e1edff37-679a-4af7-beb1-32323439e85b)

when extended in explore tab and chosen platform:
![image](https://github.com/user-attachments/assets/5a2b80eb-298f-45a3-a5a1-8477d64caeeb)

thats basically all i could find that was glitched or not rounded

**Comment by MarkChan0225** [comment]
I don't really think I can solve your problem due to difficulty. But most of the element you said is not rounded. And the expanded quest is remain unchanged because I think it looks better now and Discord is willing it to work like this.
